### DISCUSSION TITLE
Wishlist for Trans Canada Bike Route Network shown on Organic Maps

### DISCUSSION THREAD

**Discussion by BikeAcrossCanada** [organicmaps/organicmaps#4362] [Ideas]
Hi, 
I'm not a programmer, but I can help test out features.  Some of these ideas might already be in the works, others could be new.

1 - The ability to show/hide point labels.  Currently, when loading the KMZ file (bookmark) the points are shown on the map, but you need to tap on them to see them.  When zooming in or out, it would be great to be able to show/hide them.  This is especially helpful when sharing the map view.

2 - The ability to unpack the KMZ file.  Using Google Earth, I can create a KMZ file with many layers, and even have a ZIP file with a bunch of KMZ files, and they will all show up once i create a folder on the device to open it.   It would be great if it could sense that the KMZ file has many layers and show each as a bookmark rather then needing to ZIP the files.
(the Zip file is here, you can test it out if you like https://archive.org/download/TransCanadaBikeRoute/Trans%20Canada%20Bike%20Route.zip it is handy for all files to be kept in the same place with a single link)
UPDATE:  Issue: Implement bookmarks directories / subfolders #4766

3 -  Change colours for all of the things in the layer at once.  When loading different KMZ files, it would be great to be able to change the colours to differentiate them.
UPDATE: Associate a color to a group of bookmarks #1570 

4 - Change point icons for all items in the layer.  

5 - Have the 'nightmode' as a separate layer option so it's easy to find.

6 - Download all of the map areas that the KMZ file has points/tracks for all in one go.

7 - Include the category for 'Campgrounds' or specifically 'Tent Camping'.  While the category for 'RV Facilities' exists, this generally means that it's a 'RV Park' or 'Trailer Park' which doesn't always mean that tent camping is permitted.  Sometimes there is a tenting area, but if the place is listed as an 'RV Facility' it's not guaranteed.  'Campgrounds' does usually mean that RVs are permitted to camp, and it's usually for short term stays.  'RV Parks' are generally for long term stays or seasonal rental (for months at a time).
Like how the Open Campgrounds Map does have. 
https://opencampingmap.org/en/#9/46.3610/-60.4352/4/1/a40
![Screenshot_20240602-203824.png](https://github.com/organicmaps/organicmaps/assets/123879296/5a89500d-e78d-4c6c-aabd-24f87735d45c)

8 - Add an optional Grid Lines layer, as Vespucci does. (Screenshot) ![Screenshot_20240602-021544.png](https://github.com/organicmaps/organicmaps/assets/123879296/79628c76-37d7-4703-83ea-2803f58182ff)

9 - Have the ability to navigate along a loaded KMZ/GPX track. 

10 - Have a globe projection view.   
When viewing the earth zoomed out all the way, and looking at the Trans Canada Bike Route Network from a distance, you can see the curvature of the earth when looking at it in the globe projection (with Google Earth).  Google Maps doesn't use this projection, but it would be great if Organic Maps did. 
https://maplibre.org/

**Comment by biodranik** [comment]
What is (5) about?

**Reply by BikeAcrossCanada** [reply]
There are currently 2 options when you select 'Map Layers'.  Terrain and Metro.    There should be actually 4 Map Layers.  'Default' (the current map layer view when starting Organic Maps), 'Terrain', 'Metro' and 'Dark Mode'.   The 'Default' map layer should be highlighted as it's already selected by default.    Hope that helps make it clearer.

**Reply by biodranik** [reply]
You can force dark mode now from settings, right?

**Reply by BikeAcrossCanada** [reply]
Correct, from the Settings of Night Mode it can be Off/On/Auto.   The suggestion is to have Night Mode as an option in Map layers.

**Reply by biodranik** [reply]
Why do you think that it will be more useful there? Is it really helpful to switch it from layers instead of settings? Note that we'll add many other styles in layers.

**Reply by BikeAcrossCanada** [reply]
The new user for the app would expect to see the 4 map styles instead of just the 2.  As Night Mode is, in itself, a different style of map.   It can be still available in the settings as 'auto' is important, but should also be available in layers.  And good to see that more styles to choose from will be available in the future.

**Reply by biodranik** [reply]
Do you know any other map apps that show Night Mode as a selectable map layer?

**Reply by BikeAcrossCanada** [reply]
Not off hand.   Organic Maps could be the first :)
